magicctapipe/conftest.py

```diff
@@ -382,6 +384,22 @@ def dl1_lst(base_url, env_prefix):
     return LST_dl1
 
 
+@pytest.fixture(scope="session")
+def dl1_lst_old(base_url, env_prefix):
+    LST_dl1_old = []
+    for file in DL1_LST_old_lstchain:
```

Comment:
since it is only one file, and I do not think we want to add more in the future, we could have skipped the loop and define the file as simple string rather then list